File: ZayirAlkhayr.Angular/src/app/Models/shared/PDFHeaderSelected.ts
import { FilterModel } from "./FilterModel";

export interface PDFHeaderSelectedModel {
    nameEn: string;
    nameAr: string;
    isAllowSummation: boolean;
    valueType: string;
    displayOrder: number;
    isSelected: boolean;
}

export interface PDFModel {
    headers: PDFHeaderSelectedModel[];
    filterList: FilterModel[];
}

File: ZayirAlkhayr.Entities/Common/PDFHeaderSelected.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ZayirAlkhayr.Entities.Common
{
    public class PDFHeaderSelected
    {
        public string NameEn { get; set; }
        public string NameAr { get; set; }
        public bool IsAllowSummation { get; set; }
        public string ValueType { get; set; }
        public int DisplayOrder { get; set; }
    }

    public class PDFModel
    {
        public List<PDFHeaderSelected> Headers { get; set; }
        public List<FilterModel> FilterList { get; set; }
    }
}
